### PR TITLE
update version and tiup playground supports M1 silicon

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

need to cherry-pick to v5.2

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -61,10 +57,10 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
         {{< copyable "shell-regular" >}}
 
         ```shell
-        tiup playground v5.1.0 --db 2 --pd 3 --kv 3 --monitor
+        tiup playground v5.2.0 --db 2 --pd 3 --kv 3 --monitor
```
Comment (TomShawn) [inline quick-start-with-tidb.md:60]: v5.2.0 还未正式发布，这里等正式发布后再更新会不会更好？

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -20,10 +20,6 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 ## 在 Mac OS 上部署本地测试环境
 
-> **注意：**
->
-> 由于部分 TiDB 组件尚未发布支持 Apple M1 芯片的版本，暂不支持在使用 Apple M1 芯片的本地 Mac 机器上使用 `tiup playground` 命令。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:25]: 这里能否提供下自哪个版本起支持在 M1 芯片d额 macOS 上部署呢？

File: quick-start-with-tidb.md (hunk 1 above)
Comment (lilyjazz) [reply quick-start-with-tidb.md:60]: 嗯，因为今天开始支持 M1 了，我顺手更新下。预期就是 cherry-pick to v5.2，等等 v5.2

File: quick-start-with-tidb.md (hunk 2 above)
Comment (lilyjazz) [reply quick-start-with-tidb.md:25]: 这部分是删除掉的，我在下面增加了从哪个版本支持。

Comment (TomShawn) [reply quick-start-with-tidb.md:25]: 了解了

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by lilyjazz** [comment]
@TomShawn 我发现之前在 playground 启动之后的描述有一些错误，进行了修复。

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -76,6 +72,7 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
 
         > **注意：**
         >
+        > + 只有 v5.1.1 及以上版本的 TiDB，才可以在 Apple M1 Silicon 的机器上运行 `tiup playground`。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:75]: 这个实际上是从 5.1.1 开始支持的，也就是可以合并 release-5.1 文档中。那文档中的命令版本是否需要改为 5.1.1 呢？

Comment (lilyjazz) [reply quick-start-with-tidb.md:75]: 个人不建议归到 5.1 文档中，如果有人不看下面的内容，容易误以为支持所有 5.1.x，playground 并非生产环境，所以从完整的高版本开始推荐使用相对合理。

Comment (TomShawn) [reply quick-start-with-tidb.md:75]: 【不建议归到 5.1 文档中】是说不进入 release-5.1 分支文档吗？

Comment (TomShawn) [reply quick-start-with-tidb.md:75]: 那这里是否写成【只有 v5.2.0 及以上版本的 TiDB】? 否则认知起来有些混乱

Comment (lilyjazz) [reply quick-start-with-tidb.md]: 有道理，我更改了。

**Review by TomShawn — APPROVED** [review]

**Comment by xiaoyangwang5** [comment]
LGTM

**Comment by TomShawn** [comment]
@en-jin19 This PR is ready for translation. Thanks!

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -76,6 +72,7 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
 
         > **注意：**
         >
+        > + 支持 v5.2.0 及以上版本的 TiDB 在 Apple M1 Silicon 的机器上运行 `tiup playground`。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:75]:
```suggestion
        > + 支持 v5.2.0 及以上版本的 TiDB 在 Apple M1 芯片的机器上运行 `tiup playground`。
```

**Pull request merged**